content/en/security_platform/application_security/getting_started/ruby.md

```diff
@@ -17,32 +17,31 @@ further_reading:
 
 ## Get started
 
-1. **Update your gem file to include the Datadog library**:
+1. **Update your Gemfile to include the Datadog library**:
    ```
-   gem install ddtrace
+   gem 'ddtrace', '~> 1.0.0.beta1'
```

Comment:
I would add a reference+link to the [upgrade guide](https://github.com/DataDog/dd-trace-rb/blob/master/docs/UpgradeGuide.md#from-0x-to-10)